[pytest]
timeout = 240
filterwarnings =
    error
    ignore:Discarding nonzero nanoseconds
    ignore:numpy.ufunc size changed:RuntimeWarning
    ignore:can't resolve package from __spec__ or __package__, falling back on __name__ and __path__:ImportWarning
markers =
    data: tests that need data (deselect with '-m "not data"')
    pep8
    remote_data
    timeout: set test timeout (if unspecified, default is 240 seconds)
log_level = INFO
pep8ignore =
    doc/source/auto_examples/* ALL
    heliopy/data/ulysses.py E501
    heliopy/data/spice.py E501
    heliopy/data/cassini.py E501
    heliopy/spice.py E501
    examples/*.py E402
